src/Illuminate/Queue/SqsQueue.php

```diff
@@ -223,9 +223,7 @@ protected function getQueueableOptions($job, $queue): array
 
         $transformToString = fn ($value) => strval($value);
```

Comment:
`$transformToString` can be replaced by just `strval(...)` in both spots, leveraging first-class callable syntax.

Reference: https://www.php.net/manual/en/functions.first_class_callable_syntax.php